geth/proxy/transactions/sendtransaction.go

```diff
@@ -0,0 +1,315 @@
+package transactions
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"time"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/les/status"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// contains sets of possible defines the name for a giving transaction.
+const (
+	SendTransactionName = "eth_sendTransaction"
+)
+
+// ExecuteRemoteSendTransaction defines a funct
```

Comment:
Could you please improve description of this function? This is important for newcomers to understand difference between this and `ExecuteSendTransaction`.